autometa/config/project.py

```diff
@@ -197,15 +197,23 @@ def update(self, metagenome_num, fpath):
         logger.debug(f'Updated {metagenome}.config with {fpath}')
         return parse_config(old_config_fp)
 
+def main(args):
+    project = Project(args.config)
+    logger.info(f'{project.config_fpath} has {project.n_metagenomes} metagenomes in {project.dirpath}')
+    logger.info(f"metagenome config numbers: {','.join(map(str,project.metagenomes))}")
+
 if __name__ == '__main__':
-    import sys;sys.exit(1)
     import argparse
     import logging as logger
     logger.basicConfig(
-        format='%(asctime)s : %(name)s : %(levelname)s : %(message)s',
+        format='[%(asctime)s %(levelname)s] %(name)s: %(message)s',
         datefmt='%m/%d/%Y %I:%M:%S %p',
         level=logger.DEBUG)
-    parser = argparse.ArgumentParser('Autometa Project configuration')
-    parser.add_argument('--config',help='<help text of positional arg>')
+    parser = argparse.ArgumentParser(description="""
+    Contains Project class used to manipulate user's Project.
+
+    main logs status of project.
+    """)
```

Comment:
No, I think this is okay. This is part of argparse's formatter_class. By default, the lines get wrapped. But if you inspect the code,  they are visually separated. To avoid confusion, I'll remove the blank line between them.